matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java

```diff
@@ -87,6 +87,12 @@
 
     private static final String MATCHID_STRING = "matchId";
 
+    private static final SimpleDateFormat DATE_TIME_SDF = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+
+    private static final SimpleDateFormat DATE_SDF = new SimpleDateFormat("yyyy/MM/dd");
```

Comment:
I should have noticed and commented on PR #248 which added `fromDate` and `toDate`, but we should probably accept `yyyy-MM-dd` for consistency.